de.fhdo.lemma.operationdsl/src/de/fhdo/lemma/operationdsl/validation/OperationDslValidator.xtend

```diff
@@ -749,4 +749,50 @@ class OperationDslValidator extends AbstractOperationDslValidator {
             ]
         }
     }
+
+    /**
+     * Check if a used by dependency is not also included in a depends on dependency
```

Comment:
This comment does not make it that clear what is actually validated here. In fact, it should resemble the error message a bit better. What about something like "Check if node is either used by or dependent on the same node"